internal/pkg/manifest/svc.go

```diff
@@ -29,6 +29,7 @@ const (
 
 var (
 	errUnmarshalBuildOpts = errors.New("can't unmarshal build field into string or compose-style map")
+	errUnmarshalCountOpts = errors.New("can't unmarshal count field into task count or auto scaling config")
```

Comment:
nit: can we remove "can't" from the error message?

Reply:
but it's the initial error msg which isn't wrapping any other error msg. Does it make sense for users to see error msg like:
```
✘ unmarshal count field into task count or auto scaling config
```

Reply:
Oo I see, hmm I feel like this is fine but what would be really helpful to users is providing a link to our documentation explaining how they can set this field. 

Otherwise maybe this:
```suggestion
	errUnmarshalCountOpts = errors.New(`unmarshal "count" field to an integer or autoscaling configuration`)
```